### PR TITLE
Add LinearAccelerationSensor shake threshold manual test

### PR DESCRIPTION
**Pull request by xiuqijix**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-02-11 07:04:05
Finished: 2018-02-11 07:09:44


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24181)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/340059417)

File: accelerometer/LinearAccelerationSensor-threshold-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      t.done();
+    }
+  });
+
+  sensor.onerror = t.step_func_done(event => {
+    assert_unreached(event.error.name + ":" + event.error.message);
+  });
+
+  sensor.start();
+}, "Test that shake gesture along x axis of the device, threshold can greater than 25");
```
Comment (Honry) [inline accelerometer/LinearAccelerationSensor-threshold-manual.https.html]: 's/shake/when shaking'
's/threshold/the shake threshold'
's/can/can be'

Comment (kaixinjxq) [reply accelerometer/LinearAccelerationSensor-threshold-manual.https.html]: Fixed, @Honry, PTAL.

File: accelerometer/LinearAccelerationSensor-threshold-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (Honry) [inline accelerometer/LinearAccelerationSensor-threshold-manual.https.html]: >LinearAccelerationSensor-threshold-manual.https.html

The file name looks like inappropriate, how about `LinearAccelerationSensor-shake-threshold-manual.https.html`?

Comment (kaixinjxq) [reply accelerometer/LinearAccelerationSensor-threshold-manual.https.html]: Updated, @Honry, PTAL.

**Review by Honry — APPROVED** [review]
thanks, LGTM

**Pull request merged**